configuration/exodus-pipeline.yaml

```diff
@@ -532,7 +675,7 @@ Resources:
 
   CloudWatchEventRole:
     Type: AWS::IAM::Role
-    Condition: IsProd
+    Condition: NotDev
```

Comment:
I think this would need to go too if running integration tests and rollback in dev